adapter/pkg/operator/apis/dp/v1alpha1/backendpolicy_types.go

```diff
@@ -53,13 +53,26 @@ type BackendConfigs struct {
 	// +kubebuilder:validation:Enum=http;https;ws;wss
 	Protocol BackendProtocolType `json:"protocol"`
 	TLS      TLSConfig           `json:"tls,omitempty"`
+	Security SecurityConfig      `json:"security,omitempty"`
```

Comment:
this should be an array because in future there could be backend jwt or multiple backend auth support may arise.
```suggestion
	Security []SecurityConfig      `json:"security,omitempty"`
```

Reply:
resolved